src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -493,3 +496,20 @@ void CalculatorApp::GraphingCalculator::ActiveTracing_KeyUp(Windows::UI::Core::C
         args->Handled = true;
     }
 }
+
+void GraphingCalculator::GraphSettingsButton_Click(Object ^ sender, RoutedEventArgs ^ e)
+{
+    if (m_flyoutGraphSettings == nullptr)
```

Comment:
Clicking away from the flyout settings to submit your values never shows possible errors to the user, so it may look like nothing occurred. Would it make sense to have some visual indicator on the icon itself? Like an error badge or something? Maybe not?

Reply:
I've modified the behavior.

If one of the textbox in in an error state, I prevent the popup to be closed, asking the user to fix the values